enterprise/tpa_pipeline.py

```diff
@@ -78,7 +98,10 @@ def active_provider_enforces_data_sharing(request, enforcement_location):
         enforcement_location (str): the point where to see data sharing consent state.
         argument can either be "optional", 'at_login' or 'at_enrollment'
     """
-    running_pipeline = request.session.get('partial_pipeline')
+    try:
+        running_pipeline = get_partial_pipeline(request)
+    except TypeError:
+        running_pipeline = request.session.get('partial_pipeline_token')
```

Comment:
Hmm, generally code specific to tests should be kept in the `_test` files as much as possible, so that it's clear what is normal/expected behavior and what are the mocks/hacks/workarounds put in just for tests, which can be removed/refactored anytime.

In this case, can we just change the test code to mock out `get_partial_pipeline` rather than having this `try`/`except`?